src/videojs.ads.js

```diff
@@ -25,6 +25,28 @@ var
       // another cancellation is already in flight, so do nothing
       return;
     }
+
+    // Avoid content flash on iOS
+    if (videojs.browser.IS_IOS) {
+
+      // A placeholder black box will be shown in the document while the player is hidden.
+      var placeholder = document.createElement('div');
+      placeholder.style.width = player.width() + 'px';
```

Comment:
you probably want to use `player.currentWidth()` and `player.currentHeight()` here with the caveat that it will mean a new minimum supported version of videojs (5.9).